src/app.ts

```diff
@@ -155,3 +160,9 @@ function getVersion(): string {
   }
   return 'unknown';
 }
+
+function getFilesRecursive(dir: string): string[] {
+  return fs.statSync(dir).isDirectory()
+    ? [].concat(...fs.readdirSync(dir, 'utf8').map(f => getFilesRecursive(path.join(dir, f))))
+    : [dir];
```

Comment:
This is stylistic, but it seems like there is a lot happening here in a single statement.  For readability, you might consider a more traditional `if/else` clause.

Reply:
Would this be better?
```suggestion
    function getFilesRecursive(dir: string): string[] {
  if(fs.statSync(dir).isDirectory()) {
    const ancestors = fs.readdirSync(dir, 'utf8').map(f => getFilesRecursive(path.join(dir, f)));
    return [].concat(...ancestors);
  } else {
    return [dir];
  }
```

Reply:
Would this be better? 
```
  if(fs.statSync(dir).isDirectory()) {
    const ancestors = fs.readdirSync(dir, 'utf8').map(f => getFilesRecursive(path.join(dir, f)));
    return [].concat(...ancestors);
  } else {
    return [dir];
  }
```